.github/workflows/snap.yaml

```diff
@@ -0,0 +1,31 @@
+name: snap
+on:
+  push:
+    tags:
+      - '*'
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
```

Comment:
Would this kick off a new build on any new PR looking to merge into main? If the intent is for only merged commits to create a build, shouldn't the `push` section above be sufficient?

Reply:
ack, makes total sense :) totally missed that point